src/harness/fourslash.ts

```diff
@@ -1875,7 +1875,7 @@ module FourSlash {
             }
         }
 
-        private verifyProjectInfo(expected: string[]) {
+        public verifyProjectInfo(expected: string[]) {

```

Comment:
`verifyProjectInfo()` is used at https://github.com/Microsoft/TypeScript/blob/addReferencesForFourslashInTestDir/tests/cases/fourslash/fourslash.ts#L468